### DISCUSSION TITLE
Plotting marker colors with RGB tuples

### DISCUSSION THREAD

**Discussion by opierpaoli** [raphaelquast/EOmaps#202] [Q&A] [answered]
Hi, 

I recently updated my version of EOmaps from 5.3 to 7.3 using Anaconda, and I can no longer use RGB tuples as the face color when calling plot_map(). If I run the code below, I get the warning "EOmaps: Unable to plot the data for the layer 'base'!". 

Thank you.

```
from matplotlib.pyplot as plt
from eomaps import Maps
import numpy as np

x, y = np.meshgrid(np.linspace(-20, 40, 100),
                np.linspace(50, 70, 50))

r = np.random.randint(0, 100, x.shape) / 100
g = np.random.randint(0, 100, x.shape) / 100
b = [0.4]
a = np.random.randint(0, 100, x.shape) / 100

m = Maps()
m.add_feature.preset.ocean()
m.set_data(data=None, x=x, y=y)
m.plot_map(fc=(r, g, b, a))
```

**Comment by raphaelquast** [comment]
Thansk for reporting this problem! 
It seems that there is something wrong with the selection of the custom colors in relation to the data visible in the current extent.

For now the only quick fix that I can offer is to circuumvent data pre-selection based on the visible extent...
(this will make pan/zoom slower since the full dataset is plotted even if only a subset is visible, but it avoids the issue alltogether)

You can achieve this by adding the following line (before calling `m.plot_map()`)
```
m._data_manager.set_margin_factors(np.inf, np.inf)
```

Since this is an actual bug, I opened a dedicated issue for this here: [#203]
(I suggest to continue further discussions there)

I try to have a look as soon as possible!

**Comment by raphaelquast** [comment, marked as answer]
This is now fixed in EOmaps v7.3.2
